tests/testthat/test-path-subst.R

```diff
@@ -78,3 +110,43 @@ test_that("integration of path parsing works", {
                list(error = "404 - Resource Not Found"))
   expect_equal(r$route(make_req("GET", "/car/sold/true"), PlumberResponse$new()), TRUE)
 })
+
+test_that("multiple variations in path works nicely with function args detection", {
+
+  # Check when detection is not provided
+  pathDef <- "/<var0:str>/<var1:chr*>/<var2:[int]>/<var3>/<var4:*>/<var5:[*>/<var6:[]*>/<var7:[]>/<var8*>"
+  regex <- suppressWarnings(createPathRegex(pathDef))
```

Comment:
Instead of `suppressWarnings()` here and below, please use `expect_warning()` with a proper `regexp`